libraries/AC_Autorotation/AC_Autorotation.h

```diff
@@ -69,31 +104,75 @@ class AC_Autorotation
     float _vel_p;                    // Forward velocity P term.
     float _vel_ff;                   // Forward velocity Feed Forward term.
     float _accel_out;                // Acceleration value used to calculate pitch target.
+	float _entry_sink_rate;
+    float _entry_alt;
+    float _radar_alt;
+    float _flare_entry_speed;
+    float _desired_speed;
+    float _time_to_ground;
+    float _desired_sink_rate;
+    float _col_mid;
+    float _ground_clearance;
+    float _est_alt;
+    float _descent_rate_filtered;
+    float _radar_alt_prev;
+    float _radar_alt_calc;
+    float _avg_acc_z;
+    float _acc_z_sum;
+    int16_t _index;
+    float _curr_acc_z[10]{};
+    float _col_hover;
+    float _col_max;
+    float _col_min;
+    float _flare_alt_calc;
+    float _lift_hover;
+    float _c;
+    float _governed_rpm;
+    float _cushion_alt;
+    float _disc_area;
+    float _last_vertical_speed;
+    float _sink_deriv;
+    float _est_rod;
+    float _avg_sink_deriv;
+    float _avg_sink_deriv_sum;
+    int16_t _index_sink_rate;
+    float _curr_sink_deriv[20]{};
```

Comment:
```suggestion
    float _curr_sink_deriv[20];
```
Should not have to zero.

Reply:
done